contents/docs/cdp/destinations/index.md

```diff
@@ -5,129 +5,51 @@ showTitle: true
-            properties: Record<string, any>
-        }
-    }
-    project: {
-        id: number  // The ID of the PostHog project
-        name: string  // The name of the PostHog project
-        url: string  // A URL to view it in PostHog
-    }
-    source?: {
-        name: string // The name of the source (typically the destination name)
-        url: string  // A URL to view it in PostHog
-    }
-}
-```
-
-## Testing it out
-
-Once you have saved your destination, you can test it by invoking it with an example payload. You can choose whether to mock out the `async` calls (e.g. HTTP API calls) or not.
-
-> **⚠️ Warning:** If you do not mock out the API call, it will call your destination. Make sure this is definitely okay and check your example data so that you don't mess up your production data.
-
-## Advanced - custom code
-
-For most use cases, we recommend using one of the templates. These take care of a lot of the logic for you and expose simple inputs for you to configure. 
-
-You can, however, modify any destination by clicking the `show source code` option. From here you can modify the inputs (for example marking an input as secret so that it is encrypted) or the code of the function itself.
-
-The code is written in our Hog language with further documentation of available features [here](/docs/hog). The majority of destinations are simply wrappers around `fetch` - a provided function for safely doing async, retriable HTTP calls.
-
-```rust
-let res := fetch(inputs.url, {
-  'headers': inputs.headers,
-  'body': inputs.body,
-  'method': inputs.method
-});
-
-if (res.status >= 400) {
-  print('Bad response', res.status, res.body)
-}
-```
-
-### Guidelines
-
-- **Start from an existing template:** Check out existing templates that are close to your kind of use case and work from there.
-
-- **Rely on filters:** Offload as much as possible to the built in `filters` and `inputs`. This will make modifying your destination later much simpler (as well as being more performant)
-
-- **Use `inputs` wherever possible:** These are great for secrets such as API credentials. You can mark them as secret and they will not be returned to the UI in the future and will be encrypted.
-
-- **Keep it short:** We have tight controls on execution time, memory usage etc.
-
-- **Do not do more than 2 `fetch` calls:** The function will error if you do. If you have a need for more than 2 calls, please [contact support](https://us.posthog.com/#panel=support%3Asupport%3Aapps%3A%3Atrue).
-
-## FAQ
+
+As PostHog data arrives, you can export it *immediately* to other tools. You can use this for things like:
+
+- Conversion tracking
+- Monitoring key events with your chat platform
+- Enriching customer data in your CRM
+- Triggering automations
+
+PostHog enables you to send realtime data to dozens of pre-configured destinations. Check out the [data pipeline tab](https://us.posthog.com/project/2/pipeline/overview) and choose **destinations** to get started.
```

Comment:
Anytime we include a URL we should remove the "project/2" part. The UI will redirect appropriately when it loads and that way we aren't confusing things.

```suggestion
PostHog enables you to send realtime data to dozens of pre-configured destinations. Check out the [data pipeline tab](https://us.posthog.com/pipeline/overview) and choose **destinations** to get started.
```

Reply:
Oh thank you for this, that's great to know